ipxe/ipxe/build.sh

```diff
@@ -19,6 +19,10 @@ ipxe.efi)
 	cp $topdir/src/config/local/general.efi.h $topdir/src/config/local/general.h
 	make -C $topdir/src VERSION_PATCH=255 EXTRAVERSION="+ ($short_version)" bin-x86_64-efi/ipxe.efi
 	;;
+snponly.efi)
+	cp $topdir/src/config/local/general.efi.h $topdir/src/config/local/general.h
```

Comment:
this works.. for setting the general options.  Not sure if something else would be the preference here